README.md

```diff
@@ -24,6 +24,11 @@ It supports also java:
 Serialization/Deserialization of java objects:
 * The implementation needs a jackson module for serializing and deserializing the objects as described [here](https://github.com/FasterXML/jackson-modules-java8/tree/master/datetime).
 
+And it supports also .NET:
```

Comment:
In fact, this feature is realized by a still internal ApikanaGenerator, so we should not mention it here.

Reply:
Removed it.